firefox_puppeteer/api/places.py

```diff
@@ -89,13 +89,15 @@ def restore_default_bookmarks(self):
               let observer = {
                 observe: function (aSubject, aTopic, aData) {
                   Services.obs.removeObserver(observer, "bookmarks-restore-success");
+                  Services.obs.removeObserver(observer, "bookmarks-restore-failed");
 
-                  marionetteScriptFinished(true);
+                  marionetteScriptFinished(aTopic == "bookmarks-restore-success");
                 }
               };
 
               // Trigger the import of the default bookmarks
               Services.obs.addObserver(observer, "bookmarks-restore-success", false);
+              Services.obs.addObserver(observer, "bookmarks-restore-failed", false);
               BookmarkHTMLUtils.importFromURL(defaultBookmarks, true);
             """, script_timeout=10000)
         except TimeoutException:
```

Comment:
@gbrmachado it looks like you missed to address this comment. Right now a failure in resetting the bookmarks will be silent and cause failures in other tests.